src/containers/Ideas/Idea.js

```diff
@@ -21,7 +21,6 @@ import { Menu } from '../../components/Menu'
 import { Modal } from '../../components/Modal'
 import { StatusBarStyled } from '../../components/StatusBarStyled'
 import { TextSourceSans } from '../../components/TextSourceSans'
-import { DateService } from '../../services/DateService'
```

Comment:
can we delete this file? or is it still used somewhere?

Reply:
No, it can go, you are right, thanks!